f8a_worker/workers/git_stats.py

```diff
@@ -1,168 +0,0 @@
-"""Collect and compute Git statistics."""
-
-import numpy as np
```

Comment:
I think this was the only place where numpy (and scikit-learn as well) was used... we should delete superfluous dependencies from `requirements.txt`.